Projects/App/Sources/Apply/ViewModel/ApplyViewModel.swift

```diff
@@ -7,15 +7,48 @@
 //
 
 import Foundation
+import RxSwift
+import RxCocoa
```

Comment:
```suggestion
import Foundation

import RxCocoa
import RxSwift
```

1. 현재까지 사용했던 컨벤션을 생각해주세요 :)

2. RxCocoa를 왜 import하게 되었나요?

Reply:
1. 바로수정했습니다..! 
2. 처음에 RxCocoa 관련된 것을 사용하다보니 이것을 사용했는데 생각해보니 이에 대한 것을 쓰고있지않네요.! 로직변경하며 교체하지못했었던거 같습니다. 하지만 RxRelay가 RxCocoa 내부에 있어서 에러가 나지않아 인지 못했던거같습니다..!